src/api/deposit.test.js

```diff
@@ -14,7 +14,7 @@ describe('dvf.deposit', () => {
   it(`Deposits ERC20 token to user's vault`, async () => {
     mockGetConf()
     const starkPrivateKey = '100'
-    const amount = 1394
+    const amount = '1394'
```

Comment:
amount fails now if not a string? Does the UI need to change for this?

Reply:
No, both string and int amounts work but the test case will fail as it checks for type as well.